src/v2/view-builder/internals/index.js

```diff
@@ -0,0 +1,17 @@
+import BaseHeader from './BaseHeader';
+import BaseFooter from './BaseFooter';
+import BaseForm from './BaseForm';
+import BaseFormWithPolling from './BaseFormWithPolling';
+import BaseModel from './BaseModel';
+import BaseView from './BaseView';
+import FormInputFactory from './FormInputFactory';
+
+export const { addCustomButton, createCustomButtons, createIdpButtons } = FormInputFactory;
+export {
+  BaseHeader,
+  BaseFooter,
+  BaseForm,
+  BaseFormWithPolling,
+  BaseModel,
+  BaseView,
+};
```

Comment:
Isn't exporting individually (not as a whole object) preferred in order to support tree-shaking (if we were to)?

So something like this is what I was thinking -
```
export { default as BaseHeader } from './BaseHeader';
export { default as BaseFooter } from './BaseFooter';
.
.
.
```

Reply:
Great suggestion! Completely overlooked the ability to do this. Will update!